src/keystore/ram.py

```diff
@@ -328,15 +330,18 @@ async def setup_pin(self, get_word=None):
 
     async def change_pin(self):
         # get_auth_word function can generate words from part of the PIN
-        old_pin = await self.get_pin(title="First enter your old PIN code")
+        old_pin = await self.get_pin(title="First enter your old PIN code", with_cancel=True)
+        if old_pin == PinScreen.CANCEL_VALUE:
```

Comment:
With the change above in `get_value` method (returning None instead of custom value) we can check against `None` here:
```suggestion
        if old_pin is None:
```

Reply:
Tried that already: TypeError: can't convert 'NoneType' object to str implicitly